packages/graphql/tests/e2e/setup/apollo-server.ts

```diff
@@ -86,6 +103,30 @@ export class ApolloTestServer implements TestGraphQLServer {
         const server = new ApolloServer({
             schema,
             plugins: [
+                {
+                    requestDidStart() {
+                        return  Promise.resolve({
+                             didResolveOperation({ request, document }) {
+                                const estimators = neo4jGraphql.getComplexityEstimators()
+                                if(estimators.length) {
```

Comment:
I think we may have found the culprit of our formatting issues! 👀 Formatting is a bit all over the place (lines not ending in semicolons, missing spaces after `if` - can you look into this?

Reply:
I think this is fine now?